File: src/main/java/com/cartobucket/auth/routes/mappers/ScopeRequestFilterMapper.java
package com.cartobucket.auth.routes.mappers;

import com.cartobucket.auth.model.generated.ScopeRequestFilter;
import com.cartobucket.auth.model.generated.TemplateRequestFilter;

import java.util.List;
import java.util.UUID;

public class ScopeRequestFilterMapper {
    public static ScopeRequestFilter to(List<UUID> authorizationServerIds) {
        var filter = new ScopeRequestFilter();
        filter.setAuthorizationServerIds(authorizationServerIds);
        return filter;
    }
}

File: auth-api/src/main/kotlin/com/cartobucket/auth/api/server/routes/mappers/ScopeRequestFilterMapper.kt
// (C)2024
package com.cartobucket.auth.api.server.routes.mappers

import com.cartobucket.auth.api.dto.ScopeRequestFilter
import java.util.UUID

object ScopeRequestFilterMapper {
    fun to(authorizationServerIds: List<UUID>): ScopeRequestFilter =
        ScopeRequestFilter(
            authorizationServerIds = authorizationServerIds,
        )
}
